.github/workflows/build_test_linux_fedora.yaml

```diff
@@ -99,19 +101,23 @@ jobs:
       working-directory: ${{ github.workspace }}
       run: ls -l ./build/dpsim/examples/cxx/
 
+    - name: Install parallel testing
+      shell: bash
+      run: pip install pytest-xdist
+
     - name: Run pytest
       shell: bash
       working-directory: ${{ github.workspace }}
       env:
         PYTHONPATH: "${{ github.workspace }}/build"
       run: |
         cp -r python/src/dpsim build/
-        pytest -v examples/Notebooks
+        pytest -v examples/Notebooks -n auto
```

Comment:
same as above. Does this create the correct number of threads?

Reply:
For this one, we are getting a lower number (it starts 2 not 4 processes)